Bugzilla/DB/Schema.pm

```diff
@@ -1824,6 +1824,164 @@ use constant ABSTRACT_SCHEMA => {
             user_user_request_log_user_id_idx => ['user_id'],
         ],
     },
+
+    # OAuth2 Tables
+    # -------------
+
+    oauth2_client => {
+        FIELDS => [
+            id            => {TYPE => 'varchar(255)', NOTNULL => 1, PRIMARYKEY => 1},
+            description   => {TYPE => 'varchar(255)', NOTNULL => 1},
+            secret        => {TYPE => 'varchar(255)', NOTNULL => 1},
```

Comment:
I'm going to see if we can add a new bugzilla type that means "varchar but case sensitive"